Gemfile.lock

```diff
@@ -68,42 +68,23 @@ GEM
       tzinfo (~> 2.0)
```

Comment:
Are these bumps needed due to dependency issues or just for good measure?

Reply:
Part of the process we're following from publishing includes "bump other gems" (ie I think we're trying to get ahead of dependabot a bit as we bump rails).

Reply:
Oh, in fact in this commit they're dependencies of the rails update.

Reply:
That makes sense (getting ahead of Dependabot) 👍